rust/rust.bzl

```diff
@@ -34,6 +34,11 @@ load(
     _rust_clippy = "rust_clippy",
     _rust_clippy_aspect = "rust_clippy_aspect",
 )
+load(
+    "@io_bazel_rules_rust//rust:private/rust_analyzer.bzl",
```

Comment:
Can you use a workspace relative path here? `"//rust:private/rust_analyzer.bzl",`